packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -760,6 +760,65 @@ QUnit.module('Menu tests', {
         assert.notOk(submenu.option('visible'), 'submenu was closed');
     });
 
+    QUnit.test('Submenu should be closed when element loses focus', function(assert) {
+        const options = {
+            focusStateEnabled: true,
+            showFirstSubmenuMode: 'onClick',
+            items: [
+                { text: 'item 1', items: [{ text: 'item 11' }] },
+                { text: 'item 2', items: [{ text: 'item 21' }] },
+            ],
+        };
+        const menu = createMenu(options);
+        const $menuItem = $(menu.element).find(`.${DX_MENU_ITEM_CLASS}`);
+        const $subMenu = $menuItem.eq(1);
+
+        $($subMenu).trigger('dxclick');
+
+        const submenu = getSubMenuInstance($subMenu);
+
+        assert.strictEqual(submenu.option('visible'), true, 'submenu opened');
+
+        $(menu.element).trigger('focusout');
+
+        assert.strictEqual(submenu.option('visible'), false, 'submenu closed');
+    });
+
+    QUnit.test('Submenu should not be closed when submenu item clicked', function(assert) {
```

Comment:
this test is evergreen

Reply:
[d90a6b7f24312a9901977c1207f33c86766ffe5c]